dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/hooks/PreCheckSecurityOwnershipValidationHook.java

```diff
@@ -170,7 +163,7 @@ private void checkTeiTypeWriteAccess( ValidationErrorReporter reporter, String t
                 .errorCode( TrackerErrorCode.E1001 )
                 .addArg( user )
                 .addArg( trackedEntityType )
-                .build( bundle );
+                .build( bundle.getPreheat().getIdSchemes() );
```

Comment:
Pass the TEI to the method and use `reporter.addError( trackedEntity, E1001, user, trackedEntityType );`

Reply:
perfect!